docs/secure-connections/sauce-connect/system-requirements.md

```diff
@@ -15,12 +15,168 @@ Keep in mind that these are guidelines. Most environments have their own workloa
-    1. Go to **System Preferences** > **Security & Privacy** > **General**.
-    2. Under the header **Allow apps downloaded from**, select the option **App Store and identified developers**.
-* We strongly recommend using the [cURL command line](https://curl.haxx.se/download.html) or an equivalent tool to ensure that any error messages you receive are precise and actionable. If you're not familiar with the tool, [review their docs here](https://curl.se/docs) and then install cURL on your local machine.
+* We recommend using the [cURL command line](https://curl.haxx.se/download.html) or an equivalent tool to ensure that any error messages you receive are precise and actionable. If you're not familiar with the tool, [review their docs here](https://curl.se/docs) and then install cURL on your local machine.
 * For Unix-based systems, you may need to increase your open file limit if you plan to run a high number of parallel tests (i.e., `ulimit -n 8192`).
 
+
+## Supported Operating Systems
+
+### Sauce Connect v4.8.x
+
+<p><small><span className="sauceGold">Beta</span></small></p>
+
+<table>
+  <tr>
+   <td><strong>Name</strong>
+   </td>
+   <td><strong>Value</strong>
+   </td>
+   <td><strong>CPU Arch</strong>
+   </td>
+   <td><strong>Notes</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Linux
+   </td>
+   <td>Ubuntu 18.04 or higher
+        <p>Debian 9 or higher</p>
+        <p>CentOS/RHEL-7 or higher</p>
+        <p>SUSE Linux Enterprise 15.0 and higher</p>
+   </td>
+   <td>x86_6
+       <p>arm64</p>
+   </td>
+   <td>Unsupported OSes may still be able to run Sauce Connect, but they've not been tested for versions 4.8.x and higher.
+   </td>
+  </tr>
+  <tr>
+   <td>macOS
+   </td>
+   <td>macOS 10.13 (High Sierra) and higher
+   </td>
+   <td>x86_64
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>Windows
+   </td>
+   <td>Windows 10 and higher
+   </td>
+   <td>x86
+   </td>
+   <td>
+   </td>
+  </tr>
+</table>
+
+:::note macOS Catalina 10.15+ compatibility
+Enable your Mac to allow apps from the App Store and identified developers.  
+  1. Go to **System Preferences** > **Security & Privacy** > **General**.
+  2. Under **Allow apps downloaded from**, select the option **App Store and identified developers**.
+:::
+
+### Sauce Connect v4.7.x
+
+<table>
+  <tr>
+   <td><strong>Name</strong>
+   </td>
+   <td><strong>Value</strong>
+   </td>
+   <td><strong>CPU Arch</strong>
+   </td>
+   <td><strong>Notes</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Linux
+   </td>
+   <td>Ubuntu 14.04 or higher
+        <p>Debian 8 or higher</p>
+        <p>CentOS/RHEL-6 or higher</p>
+        <p>SUSE Linux Enterprise 12.0 and higher</p>
+   </td>
+   <td>x86_64
+   </td>
+   <td>Unsupported OSes may still be able to run Sauce Connect, but they've not been tested for versions 4.7.x and higher. Support has been ended for 32b.
```

Comment:
Support has been ended for 32-bit Linux.

Reply:
Fixed! Thank you